src/docbkx/chapter77-express-data-prep.xml

```diff
@@ -0,0 +1,370 @@
+                        <entry>localhost:3180</entry>
+                        <entry>yes</entry>
+                        <entry>server for name and phone cleansing (typically Suite6 installation)</entry>
+                    </row>
+                    <row>
+                        <entry>psp.architect.hostname</entry>
+                        <entry>localhost</entry>
+                        <entry>yes</entry>
+                        <entry>server for address cleansing (Mailroom Toolkit aka Satori Architect)</entry>
+                    </row>
+                    <row>
+                        <entry>psp.architect.port</entry>
+                        <entry>5150</entry>
+                        <entry>yes</entry>
+                        <entry>port for address cleansing (Mailroom Toolkit aka Satori Architect)</entry>
+                    </row>
+                    <row>
+                        <entry>psp.suppression.username</entry>
+                        <entry>username</entry>
+                        <entry>yes</entry>
+                        <entry>credentials for additional suppression services</entry>
+                    </row>
+                    <row>
+                        <entry>psp.suppression.password</entry>
+                        <entry>password</entry>
+                        <entry>yes</entry>
+                        <entry>credentials for additional suppression services</entry>
+                    </row>
+                    <row>
+                        <entry>psp.suppression.email</entry>
+                        <entry>username@domain.com</entry>
+                        <entry>yes</entry>
+                        <entry>target e-mail for suppression services notifications</entry>
+                    </row>
+                    <row>
+                        <entry>psp.outgoing.file.path</entry>
+                        <entry>C:\outgoing</entry>
+                        <entry>no</entry>
+                        <entry>directory for output data files (valid, invalid, samples)</entry>
+                    </row>
+                    <row>
+                        <entry>psp.postalpresorting.defaulttemplatename</entry>
+                        <entry>mailmark</entry>
+                        <entry>no</entry>
+                        <entry>template for presorting service</entry>
+                    </row>
+                    <row>
+                        <entry>psp.postalpresorting.defaultfilesfolder</entry>
+                        <entry>C:\presorting</entry>
+                        <entry>no</entry>
+                        <entry>directory for presorting service output (PDF report)</entry>
+                    </row>
+                </tbody>
+            </tgroup>
+        </table>
+
+        <emphasis>DataCleaner Monitor setup</emphasis>
+        <para>
+            In order to use <emphasis>hot folder functionality</emphasis> you have to configure DataCleaner Monitor
```

Comment:
Do we have documentation for hot folder?  If not should we add it ? @LosD, @arjan ? a link might be useful